packages/coclobas/coclobas.0.0.0/opam

```diff
@@ -14,7 +14,7 @@ build: [
 depends: [
   "ocamlfind" {build}
   "ocamlbuild" {build}
-  "solvuu-build" {build}
+  "solvuu-build" {build & = "0.2.0"}
   "nonstd"
   "sosa"
   "pvem_lwt_unix"
```

Comment:
it would be good to have a lower bound on cohttp here, as otherwise very old versions can get selected. The latest version is fine if thats what you've tested with.